packages/ses/README.md

```diff
@@ -144,6 +144,29 @@ still closes over the mutable counter.
 Hardening an object graph makes the surface immutable, but does not guarantee
 that methods are free of side effects.
 
+Hardening an object ensures that an attacker cannot alter the API surface of an
+object.
+Hardening "passable" data *does* ensure that a pair of attackers cannot use
+the data to communicate.
+That is, if two guests have access to the same data, they can't manipulate some
+property of the data to send signals.
```

Comment:
Not true. Remotables and Promises are both Passables and are used to communicate. Indeed, we could in the future consider hardened types arrays to be Remotables, alongside MapStores and SetStores.